catalog/db_diff_scanner.go

```diff
@@ -56,19 +55,23 @@ func NewDiffScanner(tx db.Tx, params doDiffParams) (*DiffScanner, error) {
 		Relation: relation,
 		params:   params,
 	}
+	scannerOpts := DBLineageScannerOptions{DBScannerOptions: DBScannerOptions{
+		After:            params.After,
+		AdditionalFields: prepareDiffAdditionalFields(params.AdditionalFields)},
+	}
```

Comment:
Format the struct initialize - DBScannerOptions like any other field in the struct

```suggestion
	scannerOpts := DBLineageScannerOptions{
		DBScannerOptions: DBScannerOptions{
			After:            params.After,
			AdditionalFields: prepareDiffAdditionalFields(params.AdditionalFields),
		},
	}
```